web-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@momentum-ui/web-components",
-  "version": "2.7.47",
+  "version": "2.7.49",
```

Comment:
It should be 2.7.48 , right?

Reply:
I see another PR here https://github.com/momentum-design/momentum-ui/pull/1482/files. So made it 2.7.49 to avoid conflicts..

Reply:
have you not taken the latest master in your branch. Please take latest master.

Reply:
You dont have to worry about other PR, if your PR needs to be merged and published it would be 1 more than previous. Other PR owners will update their PR accordingly.

Reply:
Done.. Updated the version